samples/system-test/spanner.test.js

```diff
@@ -842,6 +842,12 @@ describe('Spanner', () => {
 
   // restore_backup
   it('should restore database from a backup', async () => {
```

Comment:
```suggestion
  it('should restore database from a backup', async function () {
```
[Arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) does not have its own bindings to `this`

Reply:
Oops, I should've run the tests locally :( Thanks for the fix.